test/Directory.Build.props

```diff
@@ -2,7 +2,7 @@
 <Project>
   <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
   <PropertyGroup>
-    <TargetFramework>net8.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Why do we need to downgrade the TFM here?

Reply:
The `src/` projects are still `net6.0` but the tests got bumped to `8.0` which was causing some problems